src/wallet/test/fuzz/coinselection.cpp

```diff
@@ -117,6 +117,19 @@ FUZZ_TARGET(coinselection)
     if (total_balance >= target && subtract_fee_outputs && !HasErrorMsg(result_knapsack)) {
         assert(result_knapsack);
     }
+
+    std::vector<COutput> utxos;
+    std::set<std::shared_ptr<COutput>> utxos_set;
+    CreateCoins(fuzzed_data_provider, utxos, coin_params);
+    std::transform(utxos.begin(), utxos.end(),
+                 std::inserter(utxos_set, utxos_set.begin()),
+                 [](const COutput& output) {
+                   return std::make_shared<COutput>(output);
+                 });
```

Comment:
nit: To not have to do the transformation, could move all the `std::vector<COutput>` to `std::vector<std::shared_ptr<COutput>>`

Reply:
Done